Configuration/HLT/python/autoCondHLT.py

```diff
@@ -54,4 +64,14 @@ def autoCondHLT(autoCond):
            autoCond[key] = ( autoCond[val[0]] )
         else:
            autoCond[key] = ( autoCond[val[0]],) + val[1]
+#
+    for key in ('run2_mc', 'run2_data', 'run2_data_relval', 'run2_hlt', 'run2_hlt_relval'):
+        old = autoCond[key]
+        autoCond[key] = ( old, ) + l1Menus['GRun']
+
+    for key in ('run2_mc_hi', 'run2_hlt_hi'):
+        old = autoCond[key]
+        autoCond[key] = ( old, ) + l1Menus['HIon']
+
```

Comment:
@Martin-Grunewald the new GTs including the L1 uGT menu are available at #13709 (among other updates).
I am not sure why you have customized the 'run2_data' as well. 
This key should be used only for data reprocessing and not for tesing. 
I have removed relic dependencies on relval here: https://github.com/mmusich/cmssw/commit/0ef5db770b2178784d1422ad883da1345a98fbc3.